pkg/profiles/profiles.go

```diff
@@ -105,10 +105,19 @@ func ToRoute(route *sp.RouteSpec) (*pb.Route, error) {
 		}
 		rcs = append(rcs, pbRc)
 	}
-	timeout, err := time.ParseDuration(route.Timeout)
-	if err != nil {
-		log.Errorf("failed to parse duration for route %s: %s", route.Name, err)
-		timeout = DefaultRouteTimeout
+	timeout := DefaultRouteTimeout
+	if route.Timeout != "" {
+		timeout, err = time.ParseDuration(route.Timeout)
+		if err != nil {
+			log.Errorf(
+				"failed to parse duration for route '%s' in service profile '%s' in namespace '%s': %s",
+				route.Name,
+				profile.ObjectMeta.Name,
```

Comment:
I might be mistaken, but I think you can just do `profile.Name` and `profile.Namespace`.

Reply:
> I might be mistaken, but I think you can just do `profile.Name` and `profile.Namespace`.

Right - thanks!